### DISCUSSION TITLE
Teleport Issue "ERRO [WEB]: websocket: request origin not allowed by Upgrader. CheckOrigin cluster-name"

### DISCUSSION THREAD

**Discussion by ducth92** [gravitational/teleport#28626] [Q&A]
Hi all, 
I have a AD server (Window_Client-paltest-local) and clients computer (Joined AD named "Client01-paltest-local") as picture below:
![image](https://github.com/gravitational/teleport/assets/78575567/65231683-b591-40e3-ba47-e541936c8fe6)

When we setup AD join teleport (Follow the document: https://goteleport.com/docs/desktop-access/active-directory-manual/), I got this issue: 
![image](https://github.com/gravitational/teleport/assets/78575567/7553c51f-aecd-4d6b-8420-a753663d798d)

When we connect, it cannot teleport to AD server and client
![image](https://github.com/gravitational/teleport/assets/78575567/cb84d8c4-1329-446e-b4f1-7e9c3b212bbf)

(192.168.18.11 is server proxy).
This is file teleport.yaml.

Have anyone saw this issue. Please help me. Thanks all!

**Comment by webvictim** [comment]
Do you have a reverse proxy or layer 7 load balancer in front of your Teleport cluster? It seems like the request URLs that your Teleport server is receiving don't match its own idea of what the URL should be, so the connections are being blocked.

Can you also share your `/etc/teleport.yaml` config file?
